components/badge/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Badge</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+<body>
+    <div class="badge-1 badge-container">
+        <div  class="badge-relative">
+        <span>Text with a badge</span>
+        <span class="badge-absolute badge-sm badge-bold badge-blue badge-round">4</span>
+      </div>
```

Comment:
You can improve on the indentation here.
e.g.

```
<div class="badge-relative">
    <span>Text with a badge</span>
    <span class="badge-absolute badge-sm badge-bold badge-blue badge-round">4</span>
</div>
```

Reply:
okay i resolve that